test/functional/apps/visualize/_area_chart.js

```diff
@@ -62,6 +62,16 @@ bdd.describe('visualize app', function describeIndexTests() {
   bdd.describe('area charts', function indexPatternCreation() {
     var vizName1 = 'Visualization AreaChart';
 
+    bdd.it('should save and load with special characters', function pageHeader() {
+      const vizNamewithSpecialChars = vizName1 + '/?&=%';
+      return PageObjects.visualize.saveVisualization(vizNamewithSpecialChars)
+      .then(function (message) {
+        PageObjects.common.debug(`Saved viz message = ${message}`);
+        PageObjects.common.saveScreenshot('Visualize-area-chart-save-toast');
```

Comment:
This screenshot uses an identical name to the one saved in the test below. Also, I don't think there's a need to save a screenshot at all here, right @LeeDr?